courts_db/data/courts.json

```diff
@@ -25861,7 +26061,8 @@
         "regex": [
             "County Court of New York",
             "New York County Court",
-            "County Court of New York General Sessions"
+            "County Court of New York General Sessions",
+            "County Court"
```

Comment:
this feels not specific enough ... it will cause all sorts of confusion across all the states every time county. court comes up and we dont match.